api/src/main/java/org/pmiops/workbench/workspaces/WorkspacesController.java

```diff
@@ -233,6 +229,11 @@ private FirecloudWorkspace attemptFirecloudWorkspaceCreation(FirecloudWorkspaceI
 
     Timestamp now = new Timestamp(clock.instant().toEpochMilli());
     DbWorkspace dbWorkspace = new DbWorkspace();
+    // A little unintuitive but setting this here reflects the current state of the workspace
```

Comment:
It would probably be ideal here if the value were carried through on the billing project buffer table, rather than pulled from the config, since the billing account could change while projects are still in the buffer.

I think that's probably fine to just note this distinction and account for it when we're doing the free tier backfill. I'll make a note there.